api/app/models/concerns/spree/webhooks/has_webhooks.rb

```diff
@@ -9,9 +9,9 @@ module HasWebhooks
         after_update_commit(proc { queue_webhooks_requests!(event_name(:update)) })
 
         def queue_webhooks_requests!(event)
-          return if disable_spree_webhooks? || body.blank?
+          return if disable_spree_webhooks? || body(event: event).blank?
```

Comment:
You could do `return if disable_spree_webhooks? || (event_body = body(event: event)).blank?` and use event_body then.